resources/translations/en.edn

```diff
@@ -134,7 +134,7 @@
                           :returned "Returned"
                           :submitted "Applied"
                           :unknown "Unknown"}
-                 :latest-activity "Latest activity %1"
+                 :latest-activity "Latest activity: %1"
```

Comment:
The colon actually makes it stand out on the page, see screenshot below:

![image](https://user-images.githubusercontent.com/49641103/59455177-4fc43d00-8e1c-11e9-9887-b00981d5b1ba.png)

Also %1 can be removed now that it is not using text-format anymore.

Reply:
Good catch!